roles/installer/templates/statefulsets/postgres.yaml.j2

```diff
@@ -137,7 +137,15 @@ spec:
 {% if postgres_storage_class is defined %}
         storageClassName: '{{ postgres_storage_class }}'
 {% endif %}
-        resources: {{ postgres_storage_requirements }}
+{% if postgres_storage_requirements.limit or postgres_storage_requirements.requests %}
```

Comment:
I think you may need a conditional checking if `postgres_storage_requirements` only is defined first in addition to what you have.  I'm not sure, I will have to test it out.

Reply:
postgres_storage_requirements should always be defined since we have it in default